upgrade/Stage_0_Prerequisites.md

```diff
@@ -205,6 +205,10 @@ backup of Workload Manager configuration data and files is created. Once complet
    git push
    ```
 
+1. Check available space in Nexus, and free up space if needed.
```

Comment:
i thought we had this as part of prereq script already? if not, we need to add it to prereq, no more manual steps into stage 0

Reply:
It is a goss test. It was recommended to add to the prereq in https://github.com/Cray-HPE/docs-csm/pull/2073.

Reply:
so we should just remove this? as we are checking it automatically already. the clean up should go to troubleshoot part. my point is that we should stop adding any more manual steps into the happy path

Reply:
It will be removed. No need for this in the 1.3 steps